pkg/monitor/api/drop.go

```diff
@@ -81,10 +82,74 @@ var errors = map[uint8]string{
 	183: "Incorrect VNI from VTEP",
 }
 
-// DropReason prints the drop reason in a human readable string
-func DropReason(reason uint8) string {
+// The following values are defined in <linux/bpf.h>
+const (
+	BPF_FIB_LKUP_RET_BLACKHOLE = iota + 1
```

Comment:
I wouldn't bother parsing the ext error codes. Mainly, because we are risking to add lots of boilerplate. IMHO, checking the kernel src code is sufficient.

Reply:
I've removed the messages, let them rip